### PR TITLE
DBZ-3787 Upgrade binlog-client to 0.25.3

### PR DESCRIPTION
**Pull request by ani-sha**
https://issues.redhat.com/browse/DBZ-3787

### CODE REVIEW

**Comment by jpechane** [comment]
@ani-sha Thanks for the PR. Unfortunately this is not full integration test. You should take a look at other integration tests - they start database, modify it, write data and check the results so what is tested is interaction between Debezium and database too. This is more unit test that tests MySQL DDL parser. Could you please take a second look?

**Comment by gunnarmorling** [comment]
@ani-sha, @jpechane, so to make this test truly meaningful, we'd have to run the database with `--binlog-row-metadata=FULL` (see [here](https://github.com/osheroff/mysql-binlog-connector-java/pull/41/files#diff-e5eeb97142bd37696b4d844384c4826206b052dd6742ba479c416bedc806b453R68)), as otherwise the binlog client won't receive that new metadata attribute for invisible columns. My question is: do we want to simply enable this when we run MySQL via Docker, or should there be a new Maven profile for that which only would run that test?

**Comment by ani-sha** [comment]
@gunnarmorling So I facing an issue during the container setup. I think the provided `mysql.cnf` file is not getting overridden. My current container definition is: 
```
MySQLContainer<?> mySQLContainer = new MySQLContainer<>(MYSQL_DOCKER_IMAGE_NAME)
            .withDatabaseName(DATABASE.getDatabaseName())
            .withUsername("mysql")
            .withPassword("mysql")
            .withClasspathResourceMapping("/docker/conf/", "/etc/mysql", BindMode.READ_ONLY)
            .withLogConsumer(new Slf4jLogConsumer(LOGGER))
            .withExposedPorts(3306)
            .withNetworkAliases("mysql");
``` 
Is the resource mapping rightly done? The  path for `mysql.cnf` file is `debezium-connector-mysql/src/test/resources/docker/conf/mysql.cnf`

**Comment by gunnarmorling** [comment]
@ani-sha, in the example image, we [copy the mysql.conf](https://github.com/debezium/docker-images/blob/master/examples/mysql/1.7/Dockerfile#L5) file to the _/etc/mysql/conf.d/_ directory, not _etc/mysql_. Probably also better to solely map the single file instead of the entire directory, as there might be other things in that directory in the MySQL image already.

**Comment by ani-sha** [comment]
@gunnarmorling I am still seeing this error in the logs
```
ERROR 1410 (42000) at line 14: You are not allowed to create a user with GRANT   [io.debezium.connector.mysql.MySqlParserIT]
```
And there is still a connection failure even after setting port as done in `PostgresShutdownIT`. Could you please review & check if the port has been set correctly or anything else is missing?

**Comment by gunnarmorling** [comment]
Could you share the full stack trace, please?

>

**Comment by ani-sha** [comment]
> Could you share the full stack trace, please?

```
2021-09-07T11:55:22.5473866Z 2021-09-07 11:55:22,544 INFO   MySqlNumericColumnIT||test  Starting test io.debezium.connector.mysql.MySqlParserIT#parseTableWithVisibleColumns   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:24.3257512Z 2021-09-07 11:55:24,325 WARN   MySqlParserIT||test  folder-like containerPath in copyFileToContainer is deprecated, please explicitly specify a file path   [org.testcontainers.containers.GenericContainer]
2021-09-07T11:55:24.9033737Z 2021-09-07 11:55:24,889 INFO   ||  STDOUT: 2021-09-07 11:55:24+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.26-1debian10 started.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:24.9549343Z 2021-09-07 11:55:24,953 INFO   ||  STDOUT: 2021-09-07 11:55:24+00:00 [Note] [Entrypoint]: Switching to dedicated user 'mysql'   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:24.9611487Z 2021-09-07 11:55:24,959 INFO   ||  STDOUT: 2021-09-07 11:55:24+00:00 [Note] [Entrypoint]: Entrypoint script for MySQL Server 8.0.26-1debian10 started.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.0262676Z 2021-09-07 11:55:25,025 INFO   ||  STDOUT: 2021-09-07 11:55:25+00:00 [Note] [Entrypoint]: Initializing database files   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.0414273Z 2021-09-07 11:55:25,040 INFO   ||  STDERR: 2021-09-07T11:55:25.036736Z 0 [Warning] [MY-011070] [Server] 'Disabling symbolic links using --skip-symbolic-links (or equivalent) is the default. Consider not using this option as it' is deprecated and will be removed in a future release.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.0422818Z 2021-09-07 11:55:25,041 INFO   ||  STDERR: 2021-09-07T11:55:25.036750Z 0 [Warning] [MY-011068] [Server] The syntax 'expire-logs-days' is deprecated and will be removed in a future release. Please use binlog_expire_logs_seconds instead.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.0476625Z 2021-09-07 11:55:25,047 INFO   ||  STDERR: 2021-09-07T11:55:25.036822Z 0 [System] [MY-013169] [Server] /usr/sbin/mysqld (mysqld 8.0.26) initializing of server in progress as process 42   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.0564568Z 2021-09-07 11:55:25,056 INFO   ||  STDERR: 2021-09-07T11:55:25.055933Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:25.7947663Z 2021-09-07 11:55:25,794 INFO   ||  STDERR: 2021-09-07T11:55:25.793494Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:27.2980101Z 2021-09-07 11:55:27,297 INFO   ||  STDERR: 2021-09-07T11:55:27.296804Z 0 [Warning] [MY-013746] [Server] A deprecated TLS version TLSv1 is enabled for channel mysql_main   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:27.2986538Z 2021-09-07 11:55:27,298 INFO   ||  STDERR: 2021-09-07T11:55:27.297141Z 0 [Warning] [MY-013746] [Server] A deprecated TLS version TLSv1.1 is enabled for channel mysql_main   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:27.3909236Z 2021-09-07 11:55:27,390 INFO   ||  STDERR: 2021-09-07T11:55:27.390156Z 6 [Warning] [MY-010453] [Server] root@localhost is created with an empty password ! Please consider switching off the --initialize-insecure option.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.5565965Z 2021-09-07 11:55:30,555 INFO   ||  STDOUT: 2021-09-07 11:55:30+00:00 [Note] [Entrypoint]: Database files initialized   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.5571609Z 2021-09-07 11:55:30,555 INFO   ||  STDOUT: 2021-09-07 11:55:30+00:00 [Note] [Entrypoint]: Starting temporary server   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.8228015Z 2021-09-07 11:55:30,822 INFO   ||  STDERR: 2021-09-07T11:55:30.808237Z 0 [Warning] [MY-011070] [Server] 'Disabling symbolic links using --skip-symbolic-links (or equivalent) is the default. Consider not using this option as it' is deprecated and will be removed in a future release.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.8231496Z 2021-09-07 11:55:30,822 INFO   ||  STDERR: 2021-09-07T11:55:30.808259Z 0 [Warning] [MY-011068] [Server] The syntax 'expire-logs-days' is deprecated and will be removed in a future release. Please use binlog_expire_logs_seconds instead.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.8234070Z 2021-09-07 11:55:30,822 INFO   ||  STDERR: 2021-09-07T11:55:30.808397Z 0 [System] [MY-010116] [Server] /usr/sbin/mysqld (mysqld 8.0.26) starting as process 89   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:30.8255298Z 2021-09-07 11:55:30,825 INFO   ||  STDERR: 2021-09-07T11:55:30.824612Z 1 [System] [MY-013576] [InnoDB] InnoDB initialization has started.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.0048965Z 2021-09-07 11:55:31,004 INFO   ||  STDERR: 2021-09-07T11:55:31.003915Z 1 [System] [MY-013577] [InnoDB] InnoDB initialization has ended.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2540602Z 2021-09-07 11:55:31,253 INFO   ||  STDERR: 2021-09-07T11:55:31.252927Z 0 [Warning] [MY-013746] [Server] A deprecated TLS version TLSv1 is enabled for channel mysql_main   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2543959Z 2021-09-07 11:55:31,253 INFO   ||  STDERR: 2021-09-07T11:55:31.253042Z 0 [Warning] [MY-013746] [Server] A deprecated TLS version TLSv1.1 is enabled for channel mysql_main   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2554195Z 2021-09-07 11:55:31,255 INFO   ||  STDERR: 2021-09-07T11:55:31.254621Z 0 [Warning] [MY-010068] [Server] CA certificate ca.pem is self signed.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2557985Z 2021-09-07 11:55:31,255 INFO   ||  STDERR: 2021-09-07T11:55:31.255112Z 0 [System] [MY-013602] [Server] Channel mysql_main configured to support TLS. Encrypted connections are now supported for this channel.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2575420Z 2021-09-07 11:55:31,257 INFO   ||  STDERR: 2021-09-07T11:55:31.256825Z 0 [Warning] [MY-011810] [Server] Insecure configuration for --pid-file: Location '/var/run/mysqld' in the path is accessible to all OS users. Consider choosing a different directory.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2836422Z 2021-09-07 11:55:31,283 INFO   ||  STDERR: 2021-09-07T11:55:31.282967Z 0 [System] [MY-011323] [Server] X Plugin ready for connections. Socket: /var/run/mysqld/mysqlx.sock   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2851084Z 2021-09-07 11:55:31,284 INFO   ||  STDERR: 2021-09-07T11:55:31.284377Z 0 [System] [MY-010931] [Server] /usr/sbin/mysqld: ready for connections. Version: '8.0.26'  socket: '/var/run/mysqld/mysqld.sock'  port: 0  MySQL Community Server - GPL.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.2960770Z 2021-09-07 11:55:31,295 INFO   ||  STDOUT: 2021-09-07 11:55:31+00:00 [Note] [Entrypoint]: Temporary server started.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.9036713Z 2021-09-07 11:55:31,902 INFO   ||  STDERR: Warning: Unable to load '/usr/share/zoneinfo/iso3166.tab' as time zone. Skipping it.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:31.9039401Z 2021-09-07 11:55:31,903 INFO   ||  STDERR: Warning: Unable to load '/usr/share/zoneinfo/leap-seconds.list' as time zone. Skipping it.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.4292047Z 2021-09-07 11:55:33,428 INFO   ||  STDERR: Warning: Unable to load '/usr/share/zoneinfo/zone.tab' as time zone. Skipping it.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.4297412Z 2021-09-07 11:55:33,429 INFO   ||  STDERR: Warning: Unable to load '/usr/share/zoneinfo/zone1970.tab' as time zone. Skipping it.   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5084413Z 2021-09-07 11:55:33,508 INFO   ||  STDOUT: 2021-09-07 11:55:33+00:00 [Note] [Entrypoint]: Creating database mysql   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5209898Z 2021-09-07 11:55:33,520 INFO   ||  STDOUT: 2021-09-07 11:55:33+00:00 [Note] [Entrypoint]: Creating user mysql   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5346633Z 2021-09-07 11:55:33,534 INFO   ||  STDOUT: 2021-09-07 11:55:33+00:00 [Note] [Entrypoint]: Giving user mysql access to schema mysql   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5448675Z 2021-09-07 11:55:33,544 INFO   ||  STDOUT:    [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5465760Z 2021-09-07 11:55:33,546 INFO   ||  STDOUT: 2021-09-07 11:55:33+00:00 [Note] [Entrypoint]: /usr/local/bin/docker-entrypoint.sh: running /docker-entrypoint-initdb.d/inventory.sql   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:55:33.5667911Z 2021-09-07 11:55:33,566 INFO   ||  STDERR: ERROR 1410 (42000) at line 14: You are not allowed to create a user with GRANT   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:57:24.9674519Z 2021-09-07 11:57:24,962 WARN   MySqlParserIT||test  Variables cannot be used in the 'plugin.path' property, since the property is used by plugin scanning before the config providers that replace the variables are initialized. The raw value 'null' was used for plugin scanning, as opposed to the transformed value 'null', and this may cause unexpected results.   [org.apache.kafka.connect.runtime.WorkerConfig]
2021-09-07T11:57:24.9678956Z 2021-09-07 11:57:24,963 ERROR  MySqlParserIT||engine  The 'database.history.kafka.bootstrap.servers' value is invalid: A value is required   [io.debezium.connector.common.BaseSourceTask]
2021-09-07T11:57:24.9682213Z 2021-09-07 11:57:24,963 ERROR  MySqlParserIT||engine  The 'database.history.kafka.topic' value is invalid: A value is required   [io.debezium.connector.common.BaseSourceTask]
2021-09-07T11:57:24.9684881Z 2021-09-07 11:57:24,963 INFO   MySqlParserIT||engine  Stopping down connector   [io.debezium.connector.common.BaseSourceTask]
2021-09-07T11:57:24.9693686Z 2021-09-07 11:57:24,963 ERROR  MySqlParserIT||engine  Unable to initialize and start connector's task class 'io.debezium.connector.mysql.MySqlConnectorTask' with config: {connector.class=io.debezium.connector.mysql.MySqlConnector, database.history.file.filename=/home/runner/work/debezium/debezium/debezium-connector-mysql/target/data/file-db-history-column-constraints.txt, database.user=mysql, database.server.id=18765, database.server.name=myServer1, database.port=49167, offset.flush.interval.ms=0, database.ssl.mode=disabled, offset.storage.file.filename=/home/runner/work/debezium/debezium/debezium-connector-mysql/target/data/file-connector-offsets.txt, database.hostname=localhost, poll.interval.ms=10, database.*** name=testing-connector, binlog.buffer.size=10000, snapshot.mode=initial, database.include.list=mysql}   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:57:24.9712249Z org.apache.kafka.connect.errors.ConnectException: Error configuring an instance of MySqlConnectorTask; check the logs for details
2021-09-07T11:57:24.9714095Z 	at io.debezium.connector.common.BaseSourceTask.start(BaseSourceTask.java:123)
2021-09-07T11:57:24.9719777Z 	at io.debezium.embedded.EmbeddedEngine.run(EmbeddedEngine.java:759)
2021-09-07T11:57:24.9721451Z 	at io.debezium.embedded.AbstractConnectorTest.lambda$start$5(AbstractConnectorTest.java:369)
2021-09-07T11:57:24.9723345Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-09-07T11:57:24.9725078Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-09-07T11:57:24.9726218Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2021-09-07T11:57:24.9727835Z 2021-09-07 11:57:24,966 INFO   MySqlParserIT||test  Stopping the connector   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:57:24.9729593Z 2021-09-07 11:57:24,966 INFO   MySqlParserIT||test  Interrupting the engine   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:57:24.9947685Z 2021-09-07 11:57:24,993 INFO   MySqlParserIT||test  Stopping the connector   [io.debezium.connector.mysql.MySqlParserIT]
2021-09-07T11:57:24.9950417Z 2021-09-07 11:57:24,994 INFO   MySqlParserIT||test  Test io.debezium.connector.mysql.MySqlParserIT#parseTableWithVisibleColumns failed   [io.debezium.connector.mysql.MySqlParserIT]
```

**Comment by gunnarmorling** [comment]
@ani-sha, so I've pushed a commit which makes some progress, but it's still not working 100% yet. I changed the name of the variable to use underscores; while I'm not 100% sure whether that's needed, it's more inline with the other vars and what's shown in MySQL docs.

Re the grant issue, that's a bit tricky: In the [inventory.sql](https://github.com/debezium/docker-images/blob/master/examples/mysql/1.7/inventory.sql#L14) we grant privileges to a user ("mysqluser") which isn't created in that script (this was allowed in the past, implicitly creating the user then). It works for the tutorial, because there we [run the image](https://debezium.io/documentation/reference/tutorial.html#starting-mysql-database) with this user, thus creating it.

For now, I've worked around it by using that same user in this test, so to achieve the same result. But actually, the user should be created in the example file, using the password from the tutorial. This may break other usages of the image where it is started with that user and another password, but I think that's acceptable.

Note I'm still getting an privileges issue despite my change, not sure why, because that GRANT assigns all privileges. Perhaps try and do the change described above, so to see whether it addresses the issue?

In terms of the database name, I've adjusted the test so to use the "inventory" database, so to match what's used in the _inventory.sql_ file when assigning privileges. Testcontainers still will create another database ("mysql"), but this shouldn't matter.

**Comment by gunnarmorling** [comment]
@ani-sha, in case you missed it, I've pushed the commit wit my missing changes. Feel free to squash everything into a single commit of yourself. Thx.

**Comment by ani-sha** [comment]
@gunnarmorling Will adding a `RELOAD` when granting all privileges to `mysqluser` work?  
```
io.debezium.DebeziumException: java.sql.SQLSyntaxErrorException: Access denied; you need (at least one of) the RELOAD or FLUSH_TABLES privilege(s) for this operation
2021-09-08T10:23:02.0625139Z 	at io.debezium.pipeline.source.AbstractSnapshotChangeEventSource.execute(AbstractSnapshotChangeEventSource.java:79)
2021-09-08T10:23:02.0628539Z 	at io.debezium.pipeline.ChangeEventSourceCoordinator.lambda$start$0(ChangeEventSourceCoordinator.java:118)
2021-09-08T10:23:02.0630657Z 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
2021-09-08T10:23:02.0631927Z 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2021-09-08T10:23:02.0633793Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-09-08T10:23:02.0635717Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-09-08T10:23:02.0636935Z 	at java.base/java.lang.Thread.run(Thread.java:829)
```
@jpechane Any thoughts? ^^

**Comment by jpechane** [comment]
@ani-sha Please see https://github.com/debezium/debezium/blob/master/debezium-connector-mysql/src/test/docker/init/setup.sql#L9-L15

That's what we use for test. So yes before the Debezium connection you need to configure the user to have similar privileges.

**Comment by ani-sha** [comment]
@jpechane Won't the container be using this https://github.com/debezium/docker-images/blob/master/examples/mysql/1.7/inventory.sql? And this grants privileges to `mysqluser` which I have used.

**Comment by jpechane** [comment]
@ani-sha What if you use `debezium` user? Maybe some of the privileges does not fall under the `ALL` umberella.

**Comment by gunnarmorling** [comment]
>  I had to execute GRANT RELOAD ON *.* TO 'mysqluser';

@jpechane, should we add this to the _inventory.sql_ file in the example image then?

**Comment by jpechane** [comment]
@gunnarmorling No, our turotials and everything uses `debezium` user that is configured as it should.

**Comment by gunnarmorling** [comment]
@jpechane, so what exactly are you suggesting then?

**Comment by jpechane** [comment]
@gunnarmorling To use `debezium` user in the test.

**Comment by ani-sha** [comment]
>  To use `debezium` user in the test.

@jpechane It throws an error. 
```
STDERR: ERROR 1396 (HY000) at line 8: Operation CREATE USER failed for 'debezium'@'%'   [io.debezium.connector.mysql.MySqlParserIT]
```
Apart from changing the user to `debezium` do we need to provide any other grants?

**Comment by jpechane** [comment]
@ani-sha Please use
```
                .with(MySqlConnectorConfig.USER, "debezium")
                .with(MySqlConnectorConfig.PASSWORD, "dbz")
```
when configuring the connector

**Comment by gunnarmorling** [comment]
@ani-sha, are the test failures related?

**Comment by ani-sha** [comment]
@gunnarmorling SQL Server tests are unrelated. Rest checks passed. Could you review it?

**Comment by gunnarmorling** [comment]
@ani-sha, is the SQL Server failure tracked in JIra already? If not, can you please log an issue and reference it here?

File: debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlParserIT.java [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+/**
+ * Integration test for {@link MySqlConnector} using Testcontainers infrastructure for testing column constraints supported in MySQL 8.0.x.
+ */
+
+public class MySqlParserIT extends AbstractConnectorTest {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MySqlParserIT.class);
+
+    private static final String MYSQL_IMAGE = "debezium/example-mysql:1.7";
```
Comment (gunnarmorling) [inline debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlParserIT.java:37]: Can you log a follow-up issue to replace this hard-coded version with the code for looking up the latest stable version? We can only do this once 1.7 has been finalized though, IIRC. Thx!

**Review by gunnarmorling — APPROVED** [review]
LGTM, merging. Thx @ani-sha! Could you only follow-up on the two comments re logging potentially two Jira issues.

**Comment by ani-sha** [comment]
Thnx @gunnarmorling.
Logged https://issues.redhat.com/browse/DBZ-4004 & https://issues.redhat.com/browse/DBZ-4005

**Comment by gunnarmorling** [comment]
Raaah, the backport didn't quite work out. Looking into it.

**Comment by gunnarmorling** [comment]
Ok, should be sorted out now.

**Pull request merged**